pkg/sql/logictest/testdata/logic_test/geospatial

```diff
@@ -107,6 +107,19 @@ SELECT ST_AsText(p) FROM (VALUES
 POINT (1 2)
 POINT (3 4)
 
+query T
+SELECT ST_AsText(p) FROM (VALUES
+  ('POINT(200 200)'::geography),
```

Comment:
this code still applies for all geometry types.
`POINT(10000, 10000)::geometry` should still be POINT(10000, 10000).
This could should probably live in `NewGeography`, where we read the EWKB to GeomT, check all coordiantes are in range, and if they are not alter the SpatialObject.

Reply:
(also write integration tests here for geometry to ensure nothing has changed)

Reply:
Done! Good point, I wrote one to make sure.